java/server/src/org/openqa/grid/internal/Registry.java

```diff
@@ -19,421 +19,240 @@
 
 import net.jcip.annotations.ThreadSafe;
 
-import org.openqa.grid.internal.listeners.RegistrationListener;
-import org.openqa.grid.internal.listeners.SelfHealingProxy;
 import org.openqa.grid.internal.utils.configuration.GridHubConfiguration;
 import org.openqa.grid.web.Hub;
 import org.openqa.grid.web.servlet.handler.RequestHandler;
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.remote.internal.HttpClientFactory;
-import org.openqa.selenium.remote.server.log.LoggingManager;
 
 import java.util.List;
 import java.util.Set;
-import java.util.concurrent.CopyOnWriteArrayList;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.locks.Condition;
-import java.util.concurrent.locks.ReentrantLock;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 /**
- * Kernel of the grid. Keeps track of what's happening, what's free/used and assigned resources to
+ * Deprecated Kernel of the grid. Keeps track of what's happening, what's free/used and assigns resources to
  * incoming requests.
+ *
+ * @deprecated use {@link DefaultGridRegistry} instead. This class is just a proxy to it.
```

Comment:
@lukeis note: the go forward message was placed on the class

Reply:
thanks! can you put the notes in the other places you specify @deprecated, as you can see someone might not notice the other reference 😁

Reply:
Will do. Updating now